packages/web-ui/src/lab/Button/Button.docs.mdx

```diff
@@ -97,7 +97,7 @@ The variant prop controls the visual appearance of the button.
 
 Solid buttons are typically used for primary actions that require high emphasis.
 
-The solid button can be combined with the `cyan`, `red` & `green` color schemes.
+The solid button can be combined with the `cyan`, `red` & `green` colour schemes.
```

Comment:
I know these are docs, but maybe because we are talkign about a a prop we should use the `color` spelling,  I am fixing this on Figma too